### ISSUE TITLE
[BUG] cb() never called!

### ISSUE THREAD

**Issue by arlac77** [npm/cli#1824]
### Current Behavior:
npm reports internal error
```
npm ERR! cb() never called!
```

### Expected Behavior:
clean installation of node_modules

### Steps To Reproduce:
```sh
git clone https://github.com/arlac77/repository-provider
cd repository-provider
npm install
```

### Environment:

- OS: arch-linux (armv7h)
- Node: 14.11.0
- npm: 7.0.0-beta.11

**Comment by PeteProvar** [comment]
Can confirm this also happens on Mac OS Catalina

### Environment

* OS: Mac OS 10.15.6
* Node: v14.11.0
* NPM: 6.14.8

**Comment by BakungaBronson** [comment]
Facing the same issue on Windows 10

### Environment

- OS: Windows 10 Pro 
- Node: v14.12.0
- NPM: 6.14.8

**Comment by filoxo** [comment]
Encountered this when running `npx create-single-spa` with npm 6.14.4, node v13.14.0, on Linux chrx 4.16.18-galliumos x86_64.
Edit: also happens on npm 6.14.8 on v14.12.0.

**Comment by nlf** [comment]
We released v7.0.0-rc2 last Friday that includes some fixes that should resolve this issue, @arlac77 can you try updating to that version with `npm i -g npm@next-7` and see if your issue is resolved?

Those of you still on npm 6 will have to update to 7 to get the benefits from this

**Comment by arlac77** [comment]
Can confirm with 7.0.0-rc.2 th problem is gone
